### DISCUSSION TITLE
Please share good British English model

### DISCUSSION THREAD

**Discussion by omega3** [yl4579/StyleTTS2#118] [Q&A]
I listen to [audio samples](https://styletts2.github.io/) and for US English they are impressive. Is there a model equally good for British English. Can somebody share, please? I have pretty week Intel G4560 with integrated graphic (without separate  GPU), so I think have no chances at the moment to train. And I really don't understand what is this training all about.

As a point of reference I propose voices from a [dictionary](https://www.ldoceonline.com/dictionary/advantage). Please, notice that the pronunciation is different for US and British English. I also listened to voices available for coqui-ai and even when they try to imitate British accents the pronunciation of "advantage" is still in US English, at least it seems to me to be the case, although I am not British.

**Comment by yl4579** [comment]
I think the model already can capture British English to some degree, though unfortunately most speakers in LibriTTS are American speakers so the accent distributions are skewed. See https://github.com/yl4579/StyleTTS2/issues/95#issuecomment-1831053389 for example, one example shared by @WendongGan speaks in British accent, the reference audio is [here](https://vocaroo.com/19xm7Hwp0ktQ) and the synthesized result is [here](https://vocaroo.com/1hC8k5DSNa13).
